models/user.js

```diff
@@ -0,0 +1,20 @@
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  name: String,
+  username: { type: String, required: true, unique: true },
```

Comment:
Please use consistent style - if other keys display embedded objects on new lines, this should as well:
```js
username: {
  type: String,
  // etc.
}
```